tests/ecc.rs

```diff
@@ -304,3 +304,202 @@ fn mul_point_works() {
             .expect_err("circuit is not satisfied");
     }
 }
+
+#[test]
+fn assert_equal_point_works() {
```

Comment:
I suggest having only one circuit that executes the assertation, and the function is called with two points as arguments. Then, compute each variant in each test and call that circuit.

Reply:
Yeah, I'm actually just doing these changes! Was a bit too quick setting the changes as done, sorry!